File: skills/astra-toolkit/clients/go/examples/admin-list-databases.go
package main

import (
	"context"
	"encoding/json"
	"fmt"
	"log"
	"os"

	"github.com/datastax/astra-db-go/v2/astra"
	"github.com/datastax/astra-db-go/v2/astra/options"
)

func main() {
	ctx := context.Background()

	client := astra.NewClient(
		options.API().SetToken(os.Getenv("ASTRA_DB_APPLICATION_TOKEN")),
	)

	admin, err := client.Admin()

	if err != nil {
		log.Fatal(err)
	}

	databases, err := admin.ListDatabases(ctx)

	if err != nil {
		log.Fatal(err)
	}

	output, err := json.Marshal(databases)

	if err != nil {
		log.Fatal(err)
	}

	fmt.Println(string(output))
}

File: skills/astra-toolkit/clients/csharp/examples/admin-list-databases.cs
using DataStax.AstraDB.DataApi;

namespace Examples;

public class Program
{
  static async Task Main()
  {
    var client = new DataAPIClient(System.Environment.GetEnvironmentVariable("ASTRA_DB_APPLICATION_TOKEN"));

    var admin = client.GetAstraDatabasesAdmin();

    var databases = await admin.ListDatabasesAsync();
  }
}
